src/optimizer-page/scan-results/ScanResults.scss

```diff
@@ -103,7 +103,7 @@
   }
 
   .locked-links-checkbox {
-    margin-top: .45rem;
+    margin-top: .25rem;
```

Comment:
Just curious as to why this change is needed.

Reply:
to improve the alignment of the info icon with the checkbox label

**Before**
![Screenshot from 2025-05-06 11-29-58](https://github.com/user-attachments/assets/1879454e-0965-48e1-93df-51f434ab5f6a)
![Screenshot from 2025-05-06 11-29-19](https://github.com/user-attachments/assets/81814b2e-3000-4625-99f2-1c721e17e243)



**After**
![Screenshot from 2025-05-06 11-30-31](https://github.com/user-attachments/assets/117a4559-3a66-4935-bc71-bc1a12d10087)
![Screenshot from 2025-05-06 11-28-01](https://github.com/user-attachments/assets/39a3f23f-088f-4507-8f1d-afe7bc1accce)